### ISSUE TITLE
Build fails with ios9, cordova 4.1.1

### ISSUE THREAD

**Issue by rfesty** [Swrve/swrve-sdk-integrations#8]
Build after adding plugin with cordova plugin add plugins/SwrvePlugin results with multiple errors:
![capture](https://cloud.githubusercontent.com/assets/8027252/15360248/8b276f8c-1d15-11e6-8164-a97a522b75c4.PNG)

![capture1](https://cloud.githubusercontent.com/assets/8027252/15360252/8f970866-1d15-11e6-8935-bb1436b305e6.PNG)

![capture2](https://cloud.githubusercontent.com/assets/8027252/15360259/968656fe-1d15-11e6-8833-67aba9404a3c.PNG)

any ideas?

**Comment by Sergio-Mira** [comment]
It might seem we are using an old version of Cordova: https://github.com/Swrve/swrve-sdk-integrations/blob/master/PhoneGap/SwrvePhoneGapDemo/platforms/ios/cordova/version

We'll look into updating. Thanks for the heads up!

**Comment by Sergio-Mira** [comment]
@rfesty I've just merged into master changes that make it run in all versions. Let me know how it goes and reopen if needed!

**Comment by rfesty** [comment]
Thanks @Sergio-Mira, works great now

**Comment by rfesty** [comment]
Hi @Sergio-Mira,
we receive multiple errors when trying to archive the project:

ERROR ITMS-90087: "Unsupported Architectures. The executable for ****.app/Frameworks/SwrveSDK.framework contains unsupported architectures '[x86_64, i386]'."

ERROR ITMS-90362: "Invalid Info.plist value. The value for the key 'MinimumOSVersion' in bundle ****.app/Frameworks/SwrveSDK.framework is invalid. The minimum value is 8.0"
ERROR ITMS-90362: "Invalid Info.plist value. The value for the key 'MinimumOSVersion' in bundle ****.app/Frameworks/SwrveSDK.framework is invalid. The minimum value is 8.0"

ERROR ITMS-90209: "Invalid Segment Alignment. The app binary at '****.app/Frameworks/SwrveSDK.framework/SwrveSDK' does not have proper segment alignment. Try rebuilding the app with the latest Xcode version."
ERROR ITMS-90209: "Invalid Segment Alignment. The app binary at '****.app/Frameworks/SwrveSDK.framework/SwrveSDK' does not have proper segment alignment. Try rebuilding the app with the latest Xcode version."

ERROR ITMS-90125: "The binary is invalid. The encryption info in the LC_ENCRYPTION_INFO load command is either missing or invalid, or the binary is already encrypted. This binary does not seem to have been built with Apple's linker."
ERROR ITMS-90125: "The binary is invalid. The encryption info in the LC_ENCRYPTION_INFO load command is either missing or invalid, or the binary is already encrypted. This binary does not seem to have been built with Apple's linker."

**Comment by Sergio-Mira** [comment]
Hi @rfesty I can archive the demo app we bundle without any issues using Xcode 7.3.1.
Are you able to archive that project?
What Xcode version are you using?
What minimum version does your app have?
Any way you can reproduce with an empty app and you can share it with us?

Thanks!

**Comment by rfesty** [comment]
Hi @Sergio-Mira,
My mistake, we receive these errors when we try to "upload to app store".
it's Xcode 7.3, minimum version 8 
happens with demo app as well

**Comment by Sergio-Mira** [comment]
I've raised the min version of the framework we built and added a Build Script to remove any unwanted architectures to the demo, you'll need to add that to your app too. This is needed because our framework is universal and lets you dev on the simulator too, but it won't allow you to upload to the app store directly. More info on this article:
http://ikennd.ac/blog/2015/02/stripping-unwanted-architectures-from-dynamic-libraries-in-xcode/

Let me know if it works for you!
